tests/pytests/unit/modules/test_junos.py

```diff
@@ -0,0 +1,2852 @@
+            "model": "RE-VMX",
+            "status": "OK",
+            "up_time": "11 days, 23 hours, 16 minutes, 41 seconds",
+        },
+        "RE_hw_mi": False,
+        "current_re": ["re0", "master", "node", "fwdd", "member", "pfem"],
+        "domain": "englab.juniper.net",
+        "fqdn": "R1_re0.englab.juniper.net",
+        "hostname": "R1_re0",
+        "hostname_info": {"re0": "R1_re0", "re1": "R1_re01"},
+        "ifd_style": "CLASSIC",
+        "junos_info": {
+            "re0": {
+                "object": {
+                    "build": None,
+                    "major": (16, 1),
+                    "minor": "20160413_0837_aamish",
+                    "type": "I",
+                },
+                "text": "16.1I20160413_0837_aamish",
+            },
+            "re1": {
+                "object": {
+                    "build": None,
+                    "major": (16, 1),
+                    "minor": "20160413_0837_aamish",
+                    "type": "I",
+                },
+                "text": "16.1I20160413_0837_aamish",
+            },
+        },
+        "master": "RE0",
+        "model": "MX240",
+        "model_info": {"re0": "MX240", "re1": "MX240"},
+        "personality": "MX",
+        "re_info": {
+            "default": {
+                "0": {
+                    "last_reboot_reason": "0x200:normal shutdown",
+                    "mastership_state": "master",
+                    "model": "RE-VMX",
+                    "status": "OK",
+                },
+                "1": {
+                    "last_reboot_reason": "0x200:normal shutdown",
+                    "mastership_state": "backup",
+                    "model": "RE-VMX",
+                    "status": "OK",
+                },
+                "default": {
+                    "last_reboot_reason": "0x200:normal shutdown",
+                    "mastership_state": "master",
+                    "model": "RE-VMX",
+                    "status": "OK",
+                },
+            }
+        },
+        "re_master": {"default": "0"},
+        "serialnumber": "VMX4eaf",
+        "srx_cluster": None,
+        "switch_style": "BRIDGE_DOMAIN",
+        "vc_capable": False,
+        "vc_fabric": None,
+        "vc_master": None,
+        "vc_mode": None,
+        "version": "16.1I20160413_0837_aamish",
+        "version_RE0": "16.1I20160413_0837_aamish",
+        "version_RE1": "16.1I20160413_0837_aamish",
+        "version_info": {
+            "build": None,
+            "major": (16, 1),
+            "minor": "20160413_0837_aamish",
+            "type": "I",
+        },
+        "virtual": True,
+    }
+    ret["out"] = True
+    assert junos.facts() == ret
+
+
+def test_facts_exception():
+    with patch.dict(junos.__proxy__, {"junos.get_serialized_facts": raise_exception}):
+        ret = dict()
+        ret["message"] = 'Could not display facts due to "Test exception"'
+        ret["out"] = False
```

Comment:
```suggestion
        ret = dict()
        ret["message"] = 'Could not display facts due to "Test exception"'
        ret["out"] = False
```
Let's cleanup all of this prior to merging please.

Reply:
You know, it is a PITA, cleaning up after Rajvi Dhimar <rajvidhimar95@gmail.com>, the original writer of this.  I am merely porting it to pytest since have the opportunity during code-coverage.
Don't like a person's style, fine, but I am not that person.
Is there a command we can run on a file to bring it into compliance with the style desired ?, maybe have to wait for gen-AI, but given the short-handedness about to occur, rather the review be on content and not style that bugs an individual that amounts to the equivalent of a braces argument in C :laughing:

Reply:
> Don't like a person's style, fine, but I am not that person.

But you know what's proper and what is not since you touch several files in Salt every week. You don't see things like this very often.
Our job, when porting to pytest, is to also validate and cleanup, hence me mentioning this. I know it wasn't you adding this "style", it's not the first PR from you I review :grin: 

And no, there's no automated way to fix this, at least, yet.